stable/openstack-base/README.md

```diff
@@ -160,17 +135,20 @@ boot in UEFI mode:
 
 ## Configure networking
 
-Neutron networking will be configured with the aid of two scripts supplied by
+Networking will be configured with the aid of two scripts supplied by
 the bundle.
 
 > **Note**: These scripts are written for Python 2 and may not work on a modern
   system without first installing the ``python-keystoneclient`` and
   ``python-neutronclient`` deb packages.
 
+> **NOTE**: The network type required for OVN is geneve - the default for the 
+  neutron-ext-net scripts is currently gre.
+
 For the "external" network a shared router ('provider-router') will be used by
 all tenants for public access to instances. The syntax is:
 
-    ./neutron-ext-net-ksv3 --network-type flat \
+    ./neutron-ext-net-ksv3 --network-type geneve \
```

Comment:
I am not sure we need to change `flat` to `geneve`. Unless I am missing something, overlay provider networks are not the most typical use-case which requires networking hardware integration. A user of a base bundle would most likely use a `flat` or `vlan` provider network.

The OVN mechanism driver supports `geneve`, `vlan` and `flat` networks. I think 
https://opendev.org/openstack/neutron/src/commit/d444a8fbf073c96579987c8e934e7753a4150975/neutron/plugins/ml2/drivers/ovn/mech_driver/mech_driver.py#L310-L314
https://docs.openstack.org/neutron/ussuri/feature_classification/provider_network_support_matrix.html

Reply:
I think this change was meant for the internal network. Note that in the mean time we have updated the script to use Neutron's default when type not specified (see openstack-charmers/openstack-charm-testing#61), and there is also a proposal to switch to using openstack CLI commands in #171.